internal/store/users.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package store
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/mongodb/mongocli/internal/config"
+	atlas "go.mongodb.org/atlas/mongodbatlas"
+	"go.mongodb.org/ops-manager/opsmngr"
+)
+
+//go:generate mockgen -destination=../mocks/mock_users.go -package=mocks github.com/mongodb/mongocli/internal/store UserCreator
+
+type UserCreator interface {
+	CreateUser(*UserView) (interface{}, error)
+}
+
+type UserView struct {
+	opsmngr.User
+	AtlasRoles   []atlas.AtlasRole `json:"roles"`
+	MobileNumber string            `json:"mobileNumber"`
```

Comment:
I think the ops manager API accepts `mobileNumber` so we may need to add this to the go client,

Reply:
yes, you're right here. I am going to add it.

Reply:
https://github.com/mongodb/go-client-mongodb-ops-manager/pull/44